src/chronometer.js

```diff
@@ -1,33 +1,45 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = null;
   }
 
   start(printTimeCallback) {
-    // ... your code goes here
+    this.intervalId = setInterval(() => {
+      this.currentTime ++
+      if (typeof printTimeCallback === 'function') {
+        printTimeCallback()
+      }
+    }, 1000)
   }
 
   getMinutes() {
-    // ... your code goes here
+    return Math.floor(this.currentTime / 60)
   }
 
   getSeconds() {
-    // ... your code goes here
+    return Math.floor(this.currentTime % 60)
   }
 
   computeTwoDigitNumber(value) {
-    // ... your code goes here
+    if (value < 10) {
+      return '0' + value;
+    }
+    return '' + value;
   }
 
   stop() {
-    // ... your code goes here
+    return clearInterval(this.intervalId)
   }
 
   reset() {
-    // ... your code goes here
+    this.currentTime = 0
   }
 
   split() {
-    // ... your code goes here
+    return (
+      this.computeTwoDigitNumber(this.getMinutes()) + ':' +
+      this.computeTwoDigitNumber(this.getSeconds())
+    )
   }
 }
```

Comment:
Boa Phe! Apenas algumas alterações pra vc então. Nesse último, procure treinar mais os templates ` ` porque vamos usar muito mais: 
`${this.computeTwoDigitNumber(this.getMinutes())}:${this.computeTwoDigitNumber(this.getSeconds())}`